### PR TITLE
Linked Posts ORM work

### PR DESCRIPTION
**Pull request by sc0ttkclark**
https://central.tri.be/issues/116278

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Figure out logic&quot; | [Linked_Posts.php:79](https://github.com/moderntribe/the-events-calendar/blob/763604fe9857f1dfef14bc5e772278d601ffbbda/src/Tribe/Repositories/Linked_Posts.php#L79)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Repositories/Linked_Posts.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+			if ( ! $post_id ) {
+				continue;
+			}
+
+			$post_ids[] = $post_id;
+		}
+
+		$post_ids = array_unique( $post_ids );
+
+		// @todo Figure out logic.
```
Comment (sc0ttkclark) [inline src/Tribe/Repositories/Linked_Posts.php]: This is the last method still in progress

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Figure out logic&quot; | [Linked_Posts.php:79](https://github.com/moderntribe/the-events-calendar/blob/d0fbbf64a212277fccbe714eac9bae284ce12899/src/Tribe/Repositories/Linked_Posts.php#L79)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Figure out logic&quot; | [Linked_Posts.php:79](https://github.com/moderntribe/the-events-calendar/blob/d5685aa789da65c4bade5f7b8270ec2559ccf6f1/src/Tribe/Repositories/Linked_Posts.php#L79)

> via jenkins/codesniffer-phpcs

**Comment by borkweb** [comment]
After the tests are passing, I suppose.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Figure out logic&quot; | [Linked_Posts.php:79](https://github.com/moderntribe/the-events-calendar/blob/ac92c7e8a20f6dbfde2269d1cc787995fa92a31f/src/Tribe/Repositories/Linked_Posts.php#L79)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Figure out logic&quot; | [Linked_Posts.php:79](https://github.com/moderntribe/the-events-calendar/blob/421a42bf521de32a66b661c5edb617be9df2a20e/src/Tribe/Repositories/Linked_Posts.php#L79)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Figure out logic&quot; | [Linked_Posts.php:79](https://github.com/moderntribe/the-events-calendar/blob/3c28ea90f3a024b8d8bdd73016c38ddce720a042/src/Tribe/Repositories/Linked_Posts.php#L79)

> via jenkins/codesniffer-phpcs

**Pull request merged**